modules/renter/worker.go

```diff
@@ -71,12 +71,12 @@ type (
 		// priority upload chan, and finally all of the work in the upload
 		// chan.
 		//
-		// A busy higher priority channel is able to entriely starve all of the
+		// A busy higher priority channel is able to entirely starve all of the
 		// channels with lower priority.
-		downloadChan         chan downloadWork // higher priority than all uploads
-		killChan             chan struct{}     // highest priority
-		priorityDownloadChan chan downloadWork // higher priority than downloads (used for user-initiated downloads)
-		uploadChan           chan uploadWork   // lowest priority
+		downloadChan         chan downloadWork // Higher priority than all uploads.
+		killChan             chan struct{}     // Highest priority.
+		priorityDownloadChan chan downloadWork // Higher priority than downloads (used for user-initiated downloads).
+		uploadChan           chan uploadWork   // Lowest priority.
```

Comment:
we relax the conventions for "inline" comments, since they typically aren't full sentences -- they don't need capitalization or periods. If more explanation is required, it should go on top of the code rather than beside it.